rules/CodingStyle/Rector/ClassMethod/MakeInheritedMethodVisibilitySameAsParentRector.php

```diff
@@ -163,13 +167,7 @@ private function isConstructorWithStaticFactory(ClassMethod $classMethod, string
             return false;
         }
 
-        $classLike = $this->betterNodeFinder->findParentType($classMethod, Class_::class);
-        $classLike = $this->betterNodeFinder->findParentType($classMethod, Class_::class);
-        if (! $classLike instanceof Class_) {
-            return false;
-        }
-
-        foreach ($classLike->getMethods() as $iteratedClassMethod) {
+        foreach ($class->getMethods() as $iteratedClassMethod) {
```

Comment:
the class->getMethods() need to be moved up to parent loop before 1st loop so no need to call again in 2nd loop

Reply:
I've update the PR :+1: